### PR TITLE
Alert: Add test

### LINKED ISSUE #270: Write Tests for Alert

### PR DESCRIPTION
**Pull request by tingtingdisco**
Closes #270

### CODE REVIEW

File: src/General/Alert/Alert.test.tsx [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+import { PrimaryColor, SecondaryColor } from '../../Style/Colors';
+import Alert from './Alert';
+
+describe('<Alert/> should match snapshot, when isOpen is falsy', () => {
+  test('snapshot should be null regardless of alert type', () => {
+    const snapshot = renderer
+      .create(
+        <Alert
+          type={'success'}
+          message="You have successfully applied to Glints as Software Engineer."
```
Comment (ritikamotwani) [inline src/General/Alert/Alert.test.tsx:15]: Can this message be a constant as it's being used everywhere in this file?

File: src/General/Alert/Alert.test.tsx [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+    const snapshot = renderer.create(<AlertComponent type="danger" />).toJSON();
+    expect(snapshot).toMatchSnapshot();
+  });
+
+  test('type is info', () => {
+    const snapshot = renderer.create(<AlertComponent type="info" />).toJSON();
+    expect(snapshot).toMatchSnapshot();
+  });
+
+  test('type is not a predefined type', () => {
```
Comment (ritikamotwani) [inline src/General/Alert/Alert.test.tsx:59]: What's a pre-defined type?

File: src/General/Alert/Alert.test.tsx [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+    const { queryByRole } = render(<AlertComponent isOpenProp={true} />);
+    const alert = queryByRole('alertdialog');
+    fireEvent.keyDown(alert, { key: 'Esc', keyCode: 27 });
+    expect(alert).toBeInTheDocument();
+    await wait(() => {
+      expect(alert).not.toBeInTheDocument();
+    });
+  });
+
+  test('autoClose', async () => {
```
Comment (ritikamotwani) [inline src/General/Alert/Alert.test.tsx:239]: Could you please explain this part?
Also didn't understand the tests about after time out? Why should pressing the close icon close it after the time out

File: src/General/Alert/Alert.test.tsx (hunk 2 above)
Comment (tingtingdisco) [reply src/General/Alert/Alert.test.tsx:59]: success, info, warning, danger

File: src/General/Alert/Alert.test.tsx (hunk 3 above)
Comment (tingtingdisco) [reply src/General/Alert/Alert.test.tsx:239]: you can check how the component is implemented, it has a state `isVisible` to control animation and rendering. When `isOpen` changing from true to false, it sets `isVisible` to false in a 300ms timeout for running css transition.

File: src/General/Alert/Alert.test.tsx (hunk 1 above)
Comment (tingtingdisco) [reply src/General/Alert/Alert.test.tsx:15]: of course can, but it's not necessary. I think it's better to isolate test cases, and a reused variable is coupling test cases, although in this case it's ok to make it a constant.

**Review by ritikamotwani — APPROVED** [review]

**Pull request merged**